src/classes/Recipe.js

```diff
@@ -7,6 +12,37 @@ class Recipe {
     this.name = recipe.name;
     this.tags = recipe.tags;
   }
+
+  getIngredients() {
+    let currentIngredients = [];
```

Comment:
I wonder if we can implement reduce() in some way to cross-correlate this data once and then be able to access the amount, name, id, and cost?!